app/config/ApplicationConfig.scala

```diff
@@ -78,15 +74,14 @@ class ApplicationConfig @Inject()(override val runModeConfiguration: Configurati
 
   lazy val stubEnrolment: Boolean = loadBooleanConfig("enrolment.useStub")
 
-  lazy val bannerContent: Option[String] = runModeConfiguration.getString("encodedBannerContent").map(
+  lazy val bannerContent: Option[String] = configuration.getOptional[String]("encodedBannerContent").map(
     e => new String(Base64.getUrlDecoder.decode(e)))
 
-  lazy val plannedImprovementsContent: Option[String] = runModeConfiguration.getString("plannedImprovementsContent").map(e =>
+  lazy val plannedImprovementsContent: Option[String] = configuration.getOptional[String]("plannedImprovementsContent").map(e =>
     new String(Base64.getUrlDecoder.decode(e)))
 
-  override protected def mode: Mode = environment.mode
-}
+  lazy val baseUrl: String = if (Set("Dev", "Test").contains(runMode.env)) "http://localhost:9523" else ""
```

Comment:
what is this used for?

Reply:
it was there some somewhere higher up or in another class... 

redirects to localhost when run in test/dev mode... used in 5 places
![image](https://user-images.githubusercontent.com/1763911/67382863-18159080-f586-11e9-8cdb-2f11e48f01d4.png)